media-gfx/ahoviewer/ahoviewer-1.6.2.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools gnome2-utils xdg-utils
+
+DESCRIPTION="A GTK2 image viewer, manga reader, and booru browser"
+HOMEPAGE="https://github.com/ahodesuka/ahoviewer"
+SRC_URI="https://github.com/ahodesuka/ahoviewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnutls +gstreamer libsecret +rar +ssl +zip"
+
+DEPEND="
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:2.4
+	dev-libs/libconfig[cxx]
+	dev-libs/libxml2:2
+	dev-libs/libsigc++:2
+	net-misc/curl
+	gnutls? (
+		net-libs/gnutls:=
+		net-misc/curl[curl_ssl_gnutls]
+	)
+	gstreamer? (
+		media-libs/gst-plugins-bad:1.0
+		media-libs/gstreamer:1.0
+	)
+	libsecret? ( app-crypt/libsecret )
+	rar? ( app-arch/unrar )
+	ssl? (
```

Comment:
Do gnutls and ssl work simultaneously though? Otherwise I would say `gnutls? ( dep )` for gnutls and `!gnutls? ( ssl ( dep ) )` for the rest.

Reply:
or opposite `!ssl? ( gnutls? ( dep  ) )`

Reply:
>Do gnutls and ssl work simultaneously though?

There are no restrictions directly, but upstream **requires** the correct ssl curl backend, which at least limits the choice in Gentoo. 

https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-libs/efl/efl-1.21.1.ebuild#n80
I had this solution in my earlier commit, I can put it back without libressl, although it needs some `if-else` check in `src_configure`, so in my opinion the current solution (`REQUIRED_USE`) is cleaner and simpler.